src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2020 Intel Corporation.
```

Comment:
Looks like this is a new file and I guess just "2020" shall be enough.

Reply:
will do. thanks.

Reply:
done.